pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -150,6 +150,20 @@ type ScalyrConfiguration struct {
 	ScalyrMemoryLimit   string `json:"scalyr_memory_limit,omitempty"`
 }
 
+// Defines default configuration for connection pool
+type ConnectionPoolConfiguration struct {
+	NumberOfInstances    *int32 `json:"connection_pool_instances_number,omitempty"`
```

Comment:
Given we use `instancesNumber` we should use the same in the "pool" section": `numberOfInstances` although this is so long and I dont like the word instance anymore. Maybe `numberOfPods`